packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/ExportData.svelte

```diff
@@ -24,36 +18,60 @@
     },
   ]
 
+  const DELIMITERS = [
+    {
+      label: ",",
+      value: ",",
+    },
+    {
+      label: ";",
+      value: ";",
+    },
+    {
+      label: ":",
+      value: ":",
+    },
+    {
+      label: "|",
+      value: "|",
+    },
+    {
+      label: "~",
+      value: "~",
+    },
+    {
+      label: "[tab]",
+      value: "  ",
+    },
```

Comment:
`[space]` might be another relatively common one.

Reply:
Added. Also updated [tab] to use the `\t` character.